instance/models/openedx_instance.py

```diff
@@ -274,9 +274,14 @@ def get_load_balancer_configuration(self):
         """
         if not self.active_appserver:
             return self.get_preliminary_page_config(self.ref.pk)
+        ip_address = self.active_appserver.server.public_ip
+        if not ip_address:
+            # The active appserver doesn't have a public IP address.  This means that the server
+            # has been terminated, so we don't show the preliminary page in this case, and simply
+            # deconfigure the backend instead.
+            return [], []
```

Comment:
@smarnach For that instance, that might have been a manual deprovisioning from the Horizon interface - I've done this to quickly cleanup some of the unused VMs on some of the sandboxes, while waiting for https://tasks.opencraft.com/browse/OC-2041 to make it to a sprint. Since it was just sandboxes I didn't bother doing it from the console, but I missed this specific VM was actually its own instance, and it's true that it can create uncertainty in cases like this, where we might wonder if there could be an issue in our code - so I'll do it from the console next time.